### PR TITLE
add template notebook

### PR DESCRIPTION
**Pull request by drbenvincent**
Addresses #443. After this is merged, we should mention it in the Style Guide. I've made this about as minimal as possible just to get things started. Happy to take on board any suggestions.

### CODE REVIEW

File: template_notebook.ipynb [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+   "cell_type": "markdown",
+   "id": "domestic-remove",
+   "metadata": {},
+   "source": [
+    "(template_notebook)=\n",
+    "# This is a template notebook\n",
+    "\n",
+    ":::{post} January, 2023\n",
+    ":tags: binomial regression, generalized linear model, \n",
+    ":category: beginner\n",
```
Comment (OriolAbril) [inline template_notebook.ipynb:14]:
```suggestion
    ":category: beginner, reference\n",
```

File: template_notebook.ipynb [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+    "%load_ext watermark\n",
+    "%watermark -n -u -v -iv -w -p pytensor"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1e4386fc-4de9-4535-a160-d929315633ef",
+   "metadata": {},
+   "source": [
+    ":::{include} ../page_footer.md :::"
```
Comment (OriolAbril) [inline template_notebook.ipynb:156]:
```suggestion
    ":::{include} ../page_footer.md",
    ":::"
```

File: template_notebook.ipynb [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "level-balance",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "%config InlineBackend.figure_format = 'retina'\n",
```
Comment (OriolAbril) [inline template_notebook.ipynb:53]: I am not sure this is needed but I don't know what it's supposed to do either

Comment (drbenvincent) [reply template_notebook.ipynb:53]: It's not needed, but it sets the plot resolution to be high, appropriate for those with high res displays (eg on macs). I'll add a comment clarifying this, but would be ok with removing it. But preference is to keep... standard low res stuff looks bad on Apple monitors.

**Review by OriolAbril — APPROVED** [review]

**Pull request merged**